lab_1_keywords_tfidf/main.py

```diff
@@ -164,15 +164,15 @@ def calculate_tf(frequencies: dict[str, int]) -> Optional[dict[str, float]]:
 
 def calculate_tfidf(term_freq: dict[str, float], idf: dict[str, float]) -> Optional[dict[str, float]]:
     """
-    Calculates TF-IDF score for each of the tokens
-    based on its TF and IDF scores
+       Calculates TF-IDF score for each of the tokens
```

Comment:
revert this change

Reply:
done

Reply:
do you see, changes in github? This is mean^ that it is not done